src/main/java/com/emc/pravega/perf/PerfStats.java

```diff
@@ -87,9 +87,9 @@ private synchronized void record(int bytes, Instant startTime, Instant endTime)
     }
 
     private void printWindow(Instant endTime) {
-        final long elapsed = Duration.between(windowStartTime, endTime).toMillis();
-        final double recsPerSec = 1000.0 * windowCount / (double) elapsed;
-        final double mbPerSec = 1000.0 * this.windowBytes / (double) elapsed / (1024.0 * 1024.0);
+        final long elapsed = Duration.between(windowStartTime, endTime).getSeconds();
```

Comment:
Looking at the Javadoc, getSeconds() preforms a truncation to get the value in whole seconds. Given how narrow these windows are, I'm not sure we want this kind of inprecision. So let's make elapsed a double that is millis()/1000.0.